core-js/src/main/javascript/cdf/components/SelectBaseComponent.js

```diff
@@ -136,7 +136,7 @@ define([
        */
       var isEmpty = currentVals == null;
       var hasChanged = !currentIsValid;
-      if(isEmpty && this.defaultIfEmpty && firstVal != null) {
+      if(isEmpty && this.defaultIfEmpty && firstVal != null && this.useFirstValue == true) {
```

Comment:
When a user creates a CDE dashboard he will not be able to, for example, add a select component and have the first value selected because, in this case, _defaultIfEmpty && useFirstValue_ (defaults _true && undefined_) is _falsy_, and because the properties are not editable in the CDE editor, selecting the first value will not be possible.
To do this, tests, documentation and support for the properties in the CDE editor is needed for each affected component.